go.mod

```diff
@@ -1,3 +1,3 @@
 module go.yaml.in/yaml/v4
 
-go 1.16
+go 1.18
```

Comment:
I feel like you rolled back manually

Reply:
I moved it down to 1.18 because that's what I remember we had agreed to go with for v4 and I didn't want to unnecessarily rewrite all instances of `any` to `interface{}` if we are going with 1.18 at the end :-)

Reply:
done